mujoco_warp/_src/test_util.py

```diff
@@ -231,13 +232,14 @@ def benchmark(
 
     time_vec = np.zeros(nstep)
     for i in range(nstep):
-      wp.launch(
-        ctrl_noise,
-        dim=(d.nworld, m.nu),
-        inputs=[
-          m.actuator_ctrllimited, m.actuator_ctrlrange, i, 0.01
-        ],
-        outputs=[d.ctrl])  # fmt: skip
+      if ctrl_noise:
```

Comment:
I would be in favor of moving the `ctrl_noise` and `graph_conditional` changes to  a separate PR.

If you want them in this PR, let's change the `ctrl_noise` param to be a float instead of a bool that defaults to 0.01 (here and in testspeed).  if `ctrl_noise == 0.0` then you can skip the kernel launch.